packages/code-infra/src/cli/typescript.mjs

```diff
@@ -27,7 +28,8 @@ export async function emitDeclarations(tsconfig, outDir) {
     --emitDeclarationOnly
     --noEmit false
     --composite false
-    --incremental false`;
+    --incremental false
+    --declarationMap false`;
```

Comment:
Suppress sourcemap generation regardless of user settings. We can add sourcemap support in a dedicated PR if required.